src/AppBundle/Search/SearchParametersHelper.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace AppBundle\Search;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Provides a way to handle the search parameters.
+ */
+class SearchParametersHelper
```

Comment:
I'd prefer something like `SearchParametersUtils` or `SearchParametersFilter`, what do you think?